CRM/Volunteer/Hook.php

```diff
@@ -0,0 +1,13 @@
+<?php
+class CRM_Volunteer_Hook
+{
+  static $_nullObject = NULL;
+
+  static function runDefaultSettingsHook(&$defaults) {
+    return CRM_Utils_Hook::singleton()->invoke(1, $defaults, self::$_nullObject,
+      self::$_nullObject, self::$_nullObject, self::$_nullObject, self::$_nullObject,
+      'volunteer_ProjectDefaultSettings'
```

Comment:
By convention, I think this hook should be called _civicrm_volunteer_ProjectDefaultSettings_.